src/cf-layout/src/molecules/heroes.less

```diff
@@ -45,12 +40,11 @@
         .respond-to-min( @bp-sm-min, {
             .grid_column( 7, 12 );
 
-            padding: unit( @grid_gutter-width / @base-font-size-px, em )
-                     unit( 15px / @base-font-size-px, em );
+            margin: auto;
 
             // Keep the text vertically centered
-            display: table-cell;
-            vertical-align: middle;
+            //display: table-cell;
```

Comment:
I still need to do some testing in IE to see whether we need these as a backup and if so, how best to deliver them just in those situations.